AdminWebsite/AdminWebsite/ClientApp/src/app/common/custom-validations/public-holiday-validator.ts

```diff
@@ -0,0 +1,17 @@
+import { AbstractControl, ValidatorFn } from '@angular/forms';
+
+export function notPublicHolidayDateValidator(publicHolidays: Date[]): ValidatorFn {
+    return (ctrl: AbstractControl) => {
+        if (!publicHolidays.length) {
+            return null;
+        }
+        const pbTimes = publicHolidays.map(x => x.getTime());
```

Comment:
just checking getTime returns a date? or i guess they would all be midnight and it would always be a holiday?

Reply:
getTime is like timespan. the equality operator doesn't compare dates in the way we would in c#